portfoliostyle.css

```diff
@@ -0,0 +1,438 @@
+
+
+* {
+	box-sizing: border-box;
+	font-family: 'Nunito', sans-serif;
```

Comment:
Don't think this needs quotes as it's only 1 word

Reply:
I would prefer to keep the quotes in for consistency if I add fonts which have two words?